### PR TITLE
Upgrade to jakarta.ws 2.1.6

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
alluxio-bot, test this please

File: pom.xml [hunk 1]
```diff
@@ -451,7 +451,7 @@
         <artifactId>hadoop-minicluster</artifactId>
         <version>${hadoop.version}</version>
         <exclusions>
-          <!-- Excluded to avoid implementation conflict with javax.ws.rs:javax.ws.rs-api -->
+          <!-- Excluded to avoid implementation conflict with jakarta.ws.rs:jakarta.ws.rs-api -->
```
Comment (ZacBlanco) [inline pom.xml:454]: is this still required?

File: pom.xml [hunk 2]
```diff
@@ -625,7 +625,7 @@
         <artifactId>glusterfs-hadoop</artifactId>
         <version>${glusterfs-hadoop.version}</version>
         <exclusions>
-          <!-- Excluded to avoid implementation conflict with javax.ws.rs:javax.ws.rs-api -->
+          <!-- Excluded to avoid implementation conflict with jakarta.ws.rs:jakarta.ws.rs-api -->
```
Comment (ZacBlanco) [inline pom.xml:628]: is this still required?

**Comment by yuzhu** [comment]
alluxio-bot, test this please

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**